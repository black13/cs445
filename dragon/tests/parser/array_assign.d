program array_assign();
var a: array [12..1] of integer;
begin
    a[1] := 42
end.
