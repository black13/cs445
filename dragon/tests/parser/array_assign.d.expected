PROGRAM `array_assign` WITH ARGS:
AND VARIABLES:
  DECLARATIONS WITH TYPE ARRAY [`12` .. `1`] OF INTEGER:
    `a`
AND SUBPROGRAMS:
DOES:
  ASSIGN
    VARIABLE `a` INDEXED BY:
      LIT `1`
  TO
    LIT `42`
-- done dumping ast --
